### PR TITLE
refactor: removed request package, replaced underscore by lodash

### PR DESCRIPTION
**Pull request by ilin-andrey**
## Description

Changes:
* messages in slack are not used
* replaced underscore by lodash

## Checklist

- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: lib/logger.js [hunk 1]
```diff
@@ -12,9 +12,13 @@ var _util = require('./util');
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
 
 var os = require('os');
-var _ = require('underscore');
-var request = require('request');
+
+var _require = require('lodash'),
```
Comment (Dennis-Schroeder) [inline lib/logger.js:16]: typo?

Comment (ilin-andrey) [reply lib/logger.js:16]: it is auto generated code by babel =)

Comment (Dennis-Schroeder) [reply lib/logger.js:16]: ok... ugly

Comment (ilin-andrey) [reply lib/logger.js:16]: well, because we have such target js version

File: src/logger.js [hunk 2]
```diff
@@ -225,9 +208,6 @@ function logToConsole(type, sender, msgShort, _extras) {
     } catch (e) {
       logLocal('error', 'logger', 'trying to log something illegal? msgShort: ' + msgShort + ' msgLong: ' + msgLong + ' extras: ' + extras + ' orig msg: ' + e, false, { extras: extras, error: e })
     }
-    if (type === 'CRITICAL') {
```
Comment (Dennis-Schroeder) [inline src/logger.js:228]: just to point out... this is the place where we could do something like `if type CRITICAL or ERROR send to sentry` and then delete it from the backend

**Review by Dennis-Schroeder — APPROVED** [review]

File: src/logger.js (hunk 2 above)
Comment (ilin-andrey) [reply src/logger.js:228]: agree but in the current implementation of the logger it will add just problems

**Pull request merged**